src/index.js

```diff
@@ -5,19 +5,32 @@ import { createTouchPopup } from './touchPopup'
 import { renderPreview, renderLoading, renderError, renderDisambiguation, renderOffline } from './preview'
 import { getWikipediaAttrFromUrl, isTouch, getDir, isOnline } from './utils'
 
+const invokeCallback = ( events, name, params ) => {
+	const callback = events && events[ name ]
+	if ( callback instanceof Function ) {
+		try {
+			callback.apply( null, params )
+		} catch ( e ) {
+			// we don't care about buggy 3rd party functions
```

Comment:
Would the errors be accessible to them? What if we at least `console.log` them in case they care?